doc/user/content/sql/functions/now_and_mz_now.md

```diff
@@ -0,0 +1,108 @@
+---
```

Comment:
This is my first time reading this document. I know this isn't your work, but it seems very off base. Left you a number of line edits, sorry!